acoustic_msgs/msg/SonarRanges.msg

```diff
@@ -15,31 +15,7 @@ std_msgs/Header header
 # than one thing to flag beams as bad. To check if a beam is good, simply
 # check if it is zero.
 
-# Beam is OK
-uint8 BEAM_OK = 0
-# Beam is flagged as bad by the sonar.  Think hard before second-guessing
-uint8 BEAM_BAD_SONAR = 1
-# Beam is flagged as bad by some automated filter applied during processing.
-uint8 BEAM_BAD_FILTER = 2
-# Beam is flagged as bad by the user. This is probably not happening in
-# realtime, but is included here so the same message can be used throughout
-# a post-processing pipeline.
-uint8 BEAD_BAD_USER = 4
-
-# Multibeams require a sound velocity to beamform properly.  This may
-# be helpful for subsequent processing, so we include it here
-float32 sound_speed
-
-# These arrays should have the same dimensions, and should match
-# the number of beams for this sonar.
-#
-# Some fields are optional.  Those should have 0 length.
-
-# Flag values for every beam
-uint8[] flag
-
-# Truly raw multibeam data uses travel times rather than ranges.
-float32[] two_way_travel_time
+DetectionFlag[] flags
```

Comment:
Again, this comment is for a different line that Github wouldn't let me select. GRRR.

Did you intentionally leave PingInfo out of this message, or was that just an oversight?